src/hooks/vaultsHooks.ts

```diff
@@ -76,7 +79,12 @@ export const useLoadedVaultDetails = () => {
         getMegavaultHistoricalPnl(PnlTickInterval.HOUR).then(toProcessedHistoricalPnlResponse),
       ]);
       return wrapNullable(
-        VaultCalculator.calculateVaultSummary([dailyResult, hourlyResult].filter(isPresent))
+        VaultCalculator.calculateVaultSummary(
+          [dailyResult, hourlyResult].filter(isPresent),
+          megavaultHistoryStartDateMs != null && megavaultHistoryStartDateMs.length > 0
```

Comment:
is it possible to use `isTruthy` here? Should catch null, undefined, and empty strings